tools/fix_auth/fix_auth.rb

```diff
@@ -51,21 +52,13 @@ def generate_password
     end
 
     def fix_database_passwords
-      begin
-        ActiveRecord::Base.connection_config
-      rescue ActiveRecord::ConnectionNotEstablished
-        # not configured, lets try again
-        ActiveRecord::Base.logger = Logger.new("#{options[:root]}/log/fix_auth.log")
-        please_establish_connection = true
-      end
+      log_path = "#{options[:root]}/log/fix_auth.log"
       databases.each do |database|
```

Comment:
I guess I would prefer to do that in a follow up PR then have a PR that most moves code and removes functionality.